cu-calendar.sty

```diff
@@ -112,13 +112,13 @@
 }%
 %
 \def\cuDayName#1{%
-  \ifcase#1 воскресение\or%
-    понедельник\or
-    вторник\or
-    среда\or
-    четверг\or
-    пятница\or
-    суббота\fi
+  \ifcase#1 недѣ́лѧ\or%
+    понедѣ́льникъ\or
+    вто́рникъ\or
+    среда̀\or
+    четверто́къ\or
+    пѧто́къ\or
+    сꙋббѡ́та\fi
 }%
```

Comment:
And case variation is only needed for the DOW text, not for the MONTH text, right?